[3.9] bpo-41215: Don't use NULL by default in the PEG parser keyword list (GH-21355) (#21356)
python/cpython: Misc/NEWS.d/next/Core and Builtins/2020-07-06-18-36-33.bpo-41215.vFGFIz.rst

Use non-NULL default values in the PEG parser keyword list to overcome a bug that was preventing

> Python to be properly compiled when using the XLC compiler. Patch by Pablo Galindo.

Reviewer: ```suggestion
Python from being properly compiled when using the XLC compiler. Patch by Pablo Galindo.
```
